fints/client.py

```diff
@@ -605,7 +605,12 @@ def get_holdings(self, account: SEPAAccount):
 
         holdings = []
         for resp in responses:
-            mt535_lines = str.splitlines(resp.holdings)
+            if type(resp.holdings) == bytes:
+                holding_str = resp.holdings.decode('ascii')
```

Comment:
Is there a reason you are passing ``ascii`` here?

Reply:
No, i removed it.

Reply:
Thanks, but now you lost the ``:`` after the class name, that's a syntax error ;)

Reply:
Oh sorry, i have pushed a fix.